velox/serializers/PrestoSerializer.h

```diff
@@ -15,20 +15,29 @@
  */
 #pragma once
 #include "velox/common/base/Crc.h"
+#include "velox/dwio/common/Common.h"
 #include "velox/vector/VectorStream.h"
 
 namespace facebook::velox::serializer::presto {
 class PrestoVectorSerde : public VectorSerde {
  public:
   // Input options that the serializer recognizes.
   struct PrestoOptions : VectorSerde::Options {
-    explicit PrestoOptions(bool useLosslessTimestamp)
-        : useLosslessTimestamp(useLosslessTimestamp) {}
+    explicit PrestoOptions() {}
```

Comment:
Yes, then we can simplify the function, otherwise, we get nullptr Options most of the case, we should construct the PrestoOptions with its default value.
```
const PrestoVectorSerde::PrestoOptions toPrestoOptions(
    const VectorSerde::Options* options) {
  if (options == nullptr) {
    return PrestoVectorSerde::PrestoOptions();
  }
  return *(static_cast<const PrestoVectorSerde::PrestoOptions*>(options));
}
```